ios/Podfile

```diff
@@ -23,17 +23,17 @@ target 'LunesAppUI' do
   #
   # Note that if you have use_frameworks! enabled, Flipper will not work and
   # you should disable these next few lines.
-  use_flipper!
-  post_install do |installer|
-    flipper_post_install(installer)
-  end
+  # use_flipper!
+  # post_install do |installer|
+    #flipper_post_install(installer)
+  #end
```

Comment:
Why did you comment this?

Reply:
breaks the app. Will fix it later or is it really necessary to have it now?

Reply:
No, fix later is fine.